Makefile

```diff
@@ -0,0 +1,77 @@
+VERSION := $(shell date -u +"%Y%m%dT%H%MZ")
+REPO = gdc-rnaseq-cwl
+BRANCH_NAME?=unknown
+
+GIT_SHORT_HASH:=$(shell git rev-parse --short HEAD)
+COMMIT_HASH:=$(shell git rev-parse HEAD)
+
+DOCKER_REPO := quay.io/repository/ncigdc
+DOCKER_IMAGE_COMMIT := ${DOCKER_REPO}/${REPO}:${COMMIT_HASH}
+DOCKER_IMAGE_LATEST := ${DOCKER_REPO}/${REPO}:latest
+DOCKER_IMAGE := ${DOCKER_REPO}/${REPO}:${VERSION}
+
+ENTRY_WF = "./workflows/subworkflows/gdc_rnaseq_main_workflow.cwl"
+
+.PHONY: docker-*
+docker-login:
+	@echo
+	$(shell docker login -u="${SQUAY_USERNAME}" -p="${SQUAY_PASSWORD}" quay.io)
+
+.PHONY: version version-*
+version:
+	@echo --- VERSION: ${VERSION} ---
+
+version-docker:
+	@echo ${DOCKER_IMAGE_COMMIT}
+	@echo ${DOCKER_IMAGE}
+
+.PHONY: build build-* clean init init-* lint requirements run version
+init: init-pip init-hooks
+
+init-pip:
+	@echo
+	@echo -- Installing pip packages --
+	pip3 install --no-cache-dir cwltool==1.0.20180306163216
+
+init-hooks:
+	@echo
+	@echo -- Installing Precommit Hooks --
+	pre-commit install
+
+init-venv:
+	@echo
+	PIP_REQUIRE_VIRTUALENV=true pip3 install --upgrade pip-tools
+
+clean:
+	rm -rf ./build/
+	rm -rf ./dist/
+	rm -rf ./*.egg-info/
+
+.PHONY: pack
+pack:
```

Comment:
Can actually parameterize this `make pack` command to handle repos with multiple entrypoint workflows.

In this case `make pack-gdc_rnaseq` would know to pack the `gdc_rnaseq` file (once the entrypoint workflow is moved to the top level)